src/tests/test_utils/common_test_utils/include/common_test_utils/data_utils.hpp

```diff
@@ -168,12 +168,15 @@ void fill_data_roi(ov::runtime::Tensor& tensor,
 
 OPENVINO_SUPPRESS_DEPRECATED_END
 
+// k_range = start_from * k have to be integer value
+// k_start = range      * k have to be integer value
+// otherwise UB. ex 0.35 * 10 = 3.5
 template <class T>
 void inline fill_data_random(T* pointer,
                              std::size_t size,
-                             const uint32_t range = 10,
-                             double_t start_from = 0,
-                             const int32_t k = 1,
+                             const double range = 10,
```

Comment:
Why the change is required? Note that strict comparison `range == 0` won't work for floating types

Reply:
Oh... thanks. I missed that
We need to change type to support not integer intervals. E.g. [0.1, 0.2]

Reply:
Comparison with std::numeric_limits<float>::epsilon() is enough here